rtnetlink/Cargo.toml

```diff
@@ -22,6 +22,7 @@ thiserror = "1"
 netlink-packet-route = { path = "../netlink-packet-route", version = "0.5" }
 netlink-proto = { path = "../netlink-proto", version = "0.4" }
 byteordered = "0.5.0"
+nix = "0.19.0"
```

Comment:
I am not sure if this could be an issue when packaging rtnetlink on Fedora as nix seems to do not be packaged there. Introducing new dependencies can be an issue in a lot of systems due to packaging. @cathay4t

Reply:
Well without `nix` in order to do the whole unshare/mount stuff we will need to call directly the C code.
Anyway, I have tested it on Centos and it was working like a charm. 
What do you mean `nix` is not packaged for Fedora?

Reply:
Also just tested on a Fedora 33, it builds and works without any issue